lib/new_relic/agent/instrumentation/grpc/server/rpc_desc_prepend.rb

```diff
@@ -13,20 +13,20 @@ module Server
           module RpcDescPrepend
             include NewRelic::Agent::Instrumentation::GRPC::Server
 
-            def handle_request_response(*args)
-              handle_with_tracing(*args) { super }
+            def handle_request_response(active_call, mth, inter_ctx)
+              handle_with_tracing(:request_response, active_call, mth, inter_ctx) { super }
```

Comment:
- For each instrumented gRPC server request handling method, report its name to
  `issue_request_with_tracing`, and then report that name as `grpc_type`
  (thanks, Node.js agent team for the idea)